debian/preinst

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -eux
+#
+# Copyright 2025 Delphix
+#
+
+case $1 in
+upgrade)
+  # Create the working directory for the dxrt utility if it doesn't exist
+  if [ ! -d "/var/delphix/sfdc/dxrt" ]; then
+    mkdir -p "/var/delphix/sfdc/dxrt" || die "failed to mkdir /var/delphix/sfdc/dxrt"
```

Comment:
I'm confused, these commands are specific to the masking package, so why are we putting this in a masking-agnostic package, vs. putting them in the masking package?